_posts/2023-05-24-v25-rc-testing.md

```diff
@@ -21,7 +21,19 @@ commit:
   essential that the rcs are thoroughly tested. This special review club
   meeting is for people who want to help with that vital review process.
 
+- This [Bitcoin Core Release Candidate Testing
+  Guide](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/25.0-Release-Candidate-Testing-Guide) provides guidance for testing the release candidate.
 
+  - It is recommended to go through the "Preparation" steps ahead of the meeting, especially if you want to compile from source.
+
+  - Verify and confirm the successful installation of v25rc2.
```

Comment:
These feel like they're talking about the same thing, so would just merge?
```suggestion
  - It is recommended to go through the "Preparation" steps ahead of the meeting, especially if you want to compile from source. Verify and confirm the successful installation of v25rc2.
```

Reply:
Updated :+1: